content/05.dl-complexities.md

```diff
@@ -1 +1,25 @@
 ## Tip 3: Understand the complexities of training deep neural networks {#complexities}
+
+Correctly training deep neural networks is a non-trivial process.
+There are many different options and potential pitfalls at every stage.
+To get good results you have to expect to train many networks with
```

Comment:
Can you please put a single sentence on each line instead of using hard line wrapping?  That makes it easier to track and comment on each sentence.

Reply:
Sure, that makes sense. Done.